kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -149,12 +149,11 @@ grCollectionsPanel.directive('grNode', ['$parse', '$compile', function($parse, $
             // We compile the template on the fly here as angular doesn't deal
             // well with recursive templates.
             $compile(`<gr-nodes
+                ng:if="ctrl.showChildren && ctrl.node.data.children.length > 0"
```

Comment:
Putting `ng:if` first as it's the "most important" one that conditions whether the others will be applied at all or not.